124602
1.7.1
Loading dataset
{0: 126, 2: 322, 1: 239, 3: 143}
{2: 82, 1: 63, 0: 31, 3: 32}
train_dataset : batch_size -> 16, step_size -> 52, frames -> 1024
validation_dataset : batch_size -> 16, step_size -> 13, frames -> 1024
=========================================================================================================
Load model : mode -> multi, label_size -> 1, sub_class_num -> 4
train gogosing
Traceback (most recent call last):
  File "main.py", line 232, in <module>
    main()
  File "main.py", line 205, in main
    train(model, train_dataloader, epoch, criterion1, criterion2, criterion3, optimizer, writer, device, mode = params['mode'], label_num=params['label_num'], display = params['display'])
  File "/home/uchanlee/uchanlee/uchan/final_project/activate.py", line 52, in train
    for step, (inputs,text,offset, labels, age, genre, audio) in enumerate(train_dataloader):
  File "/home/uchanlee/.conda/envs/uchanpython/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 435, in __next__
    data = self._next_data()
  File "/home/uchanlee/.conda/envs/uchanpython/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 1085, in _next_data
    return self._process_data(data)
  File "/home/uchanlee/.conda/envs/uchanpython/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 1111, in _process_data
    data.reraise()
  File "/home/uchanlee/.conda/envs/uchanpython/lib/python3.6/site-packages/torch/_utils.py", line 428, in reraise
    raise self.exc_type(msg)
TypeError: Caught TypeError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/home/uchanlee/.conda/envs/uchanpython/lib/python3.6/site-packages/torch/utils/data/_utils/worker.py", line 198, in _worker_loop
    data = fetcher.fetch(index)
  File "/home/uchanlee/.conda/envs/uchanpython/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 47, in fetch
    return self.collate_fn(data)
  File "/home/uchanlee/uchanlee/uchan/final_project/UTILS/collate_batch.py", line 20, in Collate_batch
    label_list = torch.tensor(label_list, dtype=torch.int64)
TypeError: len() of unsized object

